network-api/networkapi/wagtailpages/pagemodels/publications/article.py

```diff
@@ -86,6 +88,23 @@ class ArticlePage(FoundationMetadataPageMixin, Page):
         default='image'
     )
 
+    hero_background_color = ColorField(
+        default='#ffffff',
+        help_text='Background color of the hero section.'
+        )
+
+    hero_text_color = models.CharField(
+        max_length=25,
+        choices=[
+            ('light', 'Light'),
+            ('dark', 'Dark'),
+        ],
+        default='light',
+        help_text='Color theme of hero section text. '
+                  'Choose light for light colored backgrounds '
+                  'and dark for darker color backgrounds.'
+    )
```

Comment:
**Issue:**  Find the difference between the field name and the help text very confusing. We should either name the field `hero_theme_color` or change the help text. 

I would expect the help text to say "Choose 'dark' as the text color for light colored backgrounds".

Reply:
Hi Tibor, thanks for the great feedback, I have updated the help text based on Beccas request, so I am marking this as resolved!